src/components/Pagination.svelte

```diff
@@ -0,0 +1,92 @@
+<script context="module">
+  export const paginateData = (array, totalPages) => {
+    let result = [];
+    let localData = array.slice();
+    for (let i = totalPages; i > 0; i -= 1) {
+      result.push(
+        localData.splice(0, Math.max(Math.floor(localData.length / i), 100))
+      );
+    }
+    return result;
+  };
+
+  export const makePages = (page, data, perPage = 100) => {
```

Comment:
Can you factor out the `perPage` option to a variable like `DEFAULT_ITEMS_PER_PAGE`. We should make this easier to tweak. Also, I think the default should be much smaller, like maybe 20 or so (so things appear on the screen at once)

Reply:
Updated.